pod/perlop.pod

```diff
@@ -2458,195 +2637,18 @@ producing a warning if warnings are enabled.
-
-Perl will attempt to flush all files opened for
-output before starting the child process, but this may not be supported
-on some platforms (see L<perlport>).  To be safe, you may need to set
-C<$|> (C<$AUTOFLUSH> in C<L<English>>) or call the C<autoflush()> method of
-C<L<IO::Handle>> on any open handles.
-
-Beware that some command shells may place restrictions on the length
-of the command line.  You must ensure your strings don't exceed this
-limit after any necessary interpolations.  See the platform-specific
-release notes for more details about your particular environment.
-
-Using this operator can lead to programs that are difficult to port,
-because the shell commands called vary between systems, and may in
-fact not be present at all.  As one example, the C<type> command under
-the POSIX shell is very different from the C<type> command under DOS.
-That doesn't mean you should go out of your way to avoid backticks
-when they're the right way to get something done.  Perl was made to be
-a glue language, and one of the things it glues together is commands.
-Just understand what you're getting yourself into.
-
-Like C<system>, backticks put the child process exit code in C<$?>.
-If you'd like to manually inspect failure, you can check all possible
-failure modes by inspecting C<$?> like this:
-
-    if ($? == -1) {
-        print "failed to execute: $!\n";
-    }
-    elsif ($? & 127) {
-        printf "child died with signal %d, %s coredump\n",
-            ($? & 127),  ($? & 128) ? 'with' : 'without';
-    }
-    else {
-        printf "child exited with value %d\n", $? >> 8;
-    }
-
-Use the L<open> pragma to control the I/O layers used when reading the
-output of the command, for example:
-
-    use open IN => ":encoding(UTF-8)";
-    my $x = `cmd-producing-utf-8`;
-
-C<qx//> can also be called like a function with L<perlfunc/readpipe>.
-
-See L</"I/O Operators"> for more discussion.
-
-=item C<qw/I<STRING>/>
-X<qw> X<quote, list> X<quote, words>
-
-Evaluates to a list of the words extracted out of I<STRING>, using embedded
-whitespace as the word delimiters.  It can be understood as being roughly
-equivalent to:
-
-    split(" ", q/STRING/);
-
-the differences being that it only splits on ASCII whitespace,
-generates a real list at compile time, and
-in scalar context it returns the last element in the list.  So
-this expression:
-
-    qw(foo bar baz)
-
-is semantically equivalent to the list:
-
-    "foo", "bar", "baz"
-
-Some frequently seen examples:
-
-    use POSIX qw( setlocale localeconv )
-    @EXPORT = qw( foo bar baz );
-
-A common mistake is to try to separate the words with commas or to
-put comments into a multi-line C<qw>-string.  For this reason, the
-S<C<use warnings>> pragma and the B<-w> switch (that is, the C<$^W> variable)
-produces warnings if the I<STRING> contains the C<","> or the C<"#"> character.
+=head3 Transliteration Quote-Like Operators
 
 =item C<tr/I<SEARCHLIST>/I<REPLACEMENTLIST>/cdsr>
```

Comment:
And a new =over to start this ordered list

Reply:
Likewise